packages/wallet-sdk/src/store/store.ts

```diff
@@ -135,6 +155,7 @@ export const sdkstore = createStore(
           account: state.account,
           subAccount: state.subAccount,
           config: state.config,
+          walletConnectResponse: state.walletConnectResponse,
```

Comment:
This will contain quite a bit of duplicate info in the store. I wonder if we should keep the store more generic and then rebuild the response to serve up if available.

@fan-zhang-sv what are your thoughts?

Reply:
that's fair. my only push back would be that now the SDK needs to know how to build a wallet_connect response, which isn't really its job and since storage is cheap the duplication doesn't _really_ matter

not a strong opinion though. will update